firmware/examples/PWM/source/main.cpp

```diff
@@ -0,0 +1,37 @@
+#include <cstdarg>
+#include <cstdio>
+#include <cstdint>
+#include "L0_LowLevel/delay.hpp"
+#include "L1_Drivers/gpio.hpp"
+#include "L1_Drivers/pwm.hpp"
+#include "L2_Utilities/debug_print.hpp"
+
+int main(void)
+{
+    DEBUG_PRINT("Pwm Application Starting...\n");
+
+    // Standard gpio powered LED
+    DEBUG_PRINT("Creating Gpio powered LED...\n");
+    Gpio p4_28(4,28);
+    p4_28.SetAsInput();
```

Comment:
If you are setting up a Gpio to power on an LED, doesn't it need to be an output?